lib/mondo/card.rb

```diff
@@ -0,0 +1,26 @@
+module Mondo
+  class Card < Resource
+    attr_accessor :id, :processor_token, :processor, :account_id,
+                :last_digits, :name, :expires, :status
+
+    date_accessor :created
+
+    def active?
+      status == 'ACTIVE'
+    end
+
+    def freeze!
```

Comment:
I'm of the school of thought that says `!` means "this method is the “dangerous” version of an otherwise equivalent method, with the same name minus the !"

Ie, unless there's the same method without a `!`, don't use `!`